dali/base/dasess.cpp

```diff
@@ -525,11 +519,18 @@ class CSessionRequestServer: public Thread
                 int role=0;
                 if (mb.length()-mb.getPos()>=sizeof(role)) { // a capability block present
                     mb.read(role);
-                    if (!manager.authorizeConnection(role,false)) {
+                    if (!manager.authorizeConnection(node, (DaliClientRole) role))
+                    {
+                        MilliSleep(100+getRandom()%1000); // Causes client to 'work' for a short time.
```

Comment:
Previous code was never actually executed, so doesn't really set any kind of precedent.

I wanted a delay to prevent repeated rapid requests. We can put a delay each side of the reply if you prefer.